docs/content/Configuration/Connecting-to-the-Database.md

```diff
@@ -279,6 +288,82 @@ To connect to a Elasticsearch database, use `CUBEJS_DB_URL` with the username
 and password embedded in the URL, if required. If you're not using Elastic
 Cloud, you **must** specify `CUBEJS_DB_ELASTIC_QUERY_FORMAT`.
 
+### AWS Redshift
+
+#### Configuring an export bucket
+
+<!-- prettier-ignore-start -->
+[[warning |]]
+| AWS Redshift only supports using AWS S3 for export buckets.
+<!-- prettier-ignore-end -->
+
+##### AWS S3
+
+For [improved pre-aggregation performance with large
+datasets][ref-caching-large-preaggs], enable the export bucket functionality by
+configuring Cube.js with the following environment variables:
+
+<!-- prettier-ignore-start -->
+[[info |]]
+| Ensure the AWS credentials are correctly configured in IAM to allow reads and
+| writes to the export bucket.
+<!-- prettier-ignore-end -->
+
+```dotenv
+CUBEJS_DB_EXPORT_BUCKET=my.bucket.on.s3
+CUBEJS_DB_EXPORT_BUCKET_AWS_KEY=<AWS_KEY>
+CUBEJS_DB_EXPORT_BUCKET_AWS_SECRET=<AWS_SECRET>
+CUBEJS_DB_EXPORT_BUCKET_AWS_REGION=<AWS_REGION>
+```
+
+### Snowflake
+
+#### Configuring an export bucket
+
+Snowflake supports using both AWS S3 and Google Cloud Storage for export bucket
+functionality.
+
+##### AWS S3
+
+<!-- prettier-ignore-start -->
+[[info |]]
+| Ensure the AWS credentials are correctly configured in IAM to allow reads and
+| writes to the export bucket.
+<!-- prettier-ignore-end -->
+
+```dotenv
+CUBEJS_DB_EXPORT_BUCKET=my.bucket.on.s3
+CUBEJS_DB_EXPORT_BUCKET_AWS_KEY=<AWS_KEY>
+CUBEJS_DB_EXPORT_BUCKET_AWS_SECRET=<AWS_SECRET>
+CUBEJS_DB_EXPORT_BUCKET_AWS_REGION=<AWS_REGION>
+```
+
+##### Google Cloud Storage
+
+Before configuring Cube.js, an integration must be created in Snowflake:
+
+```sql
+CREATE OR REPLACE STORAGE INTEGRATION gcs_int
+  type = external_stage
+  storage_provider = gcs
+  enabled = true
+  storage_allowed_locations = ('gcs://snowflake-export-bucket/*', 'gcs://snowflake-export-bucket/');
+GRANT USAGE ON INTEGRATION gcs_int TO ROLE SYSADMIN;
+```
```

Comment:
I think we should write about granting permissions for Snowflake:

1. `DESC INTEGRATION gcs_int;`
2. Add Snowflake's user (value of `STORAGE_GCP_SERVICE_ACCOUNT`) to bucket

Reply:
I meant to add a link to this: https://docs.snowflake.com/en/user-guide/data-load-gcs-config.html

@keydunov: Is it worth replicating the steps from the link above, or just linking out to it and then just go over the Cube.js-related parts?